CodeGen/docker/xeon/docker_compose.yaml

```diff
@@ -66,6 +66,18 @@ services:
       - BASIC_URL=${BACKEND_SERVICE_ENDPOINT}
     ipc: host
     restart: always
+  codegen-xeon-react-ui-server:
+    image: opea/codegen-react-ui:latest
+    container_name: codegen-xeon-react-ui-server
+    depends_on:
+      - codegen-xeon-backend-server
+    ports:
+      - "5174:80"
+    build:
```

Comment:
suggest to remove the build part.

Reply:
build part is needed as we cannot inject environmental values to react once the image is built